src/js/components/mapWidgets/widgetContent/coordinatesDMSSection.tsx

```diff
@@ -7,9 +7,9 @@ import { ReactComponent as TrashCanIcon } from 'images/trashCanIcon.svg';
 import 'css/coordinatesForm';
 
 interface CoordinateProps {
-  degree: number;
-  minutes: number;
-  seconds: number;
+  degree: string;
+  minutes: string;
+  seconds: string;
```

Comment:
let's create a shared interface in our interface folder called Coordinates or something along those lines and use it in `coordinatesDMSSection.tsx` and `coordinatesForm.tsx`.